src/plugins/intel_npu/src/plugin/npuw/util.hpp

```diff
@@ -14,7 +14,7 @@ namespace ov {
 namespace npuw {
 namespace util {
 
-bool is_set(const std::size_t sub_idx, const std::string& opt);
+bool is_set(const std::size_t sub_idx, const std::string& opt, const std::size_t last_idx = SIZE_MAX);
```

Comment:
please note that `NPUW_PMM` option also uses this function also to check if its value corresponds to axis: https://github.com/openvinotoolkit/openvino/blob/b7d1f1d3aa3d9a1f385f2fff2d2e078168aed25f/src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/opt.cpp#L721. Could you please consult with Dmitry if it is okay to allow this option to accept "LAST"? Might be we should `is_set()` to 2 different functions other way.

Reply:
Could I please also ask to update documentation for this options here: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_npu/src/al/include/npuw_private_properties.hpp?

Each option, that can accept "YES"/"NO" and  indices can be updated: `NPUW_DUMP_SUBS`, `NPUW_DUMP_SUBS_ON_FAIL`, `NPUW_DUMP_IO` and possibly `NPUW_PMM` also

Reply:
From my understanding for `NPUW_PMM` option `last` keyword doesn't make sense. @dmatveev could you please comment on that?

Because of that default value for this argument was added. So only if `last_idx `argument was provided in `is_set()` function (and it's not true for case with NPUW_PMM), we'll process condition check for last keyword.
https://github.com/openvinotoolkit/openvino/pull/26841/files#diff-531168f120f0773dea944034f2dff35f303d0821be290eec72eb89c311716764R38-R39

Reply:
Sure, i'll update documentation, thanks for noticing this!